taskchampion/src/storage/sqlite.rs

```diff
@@ -107,17 +107,11 @@ impl SqliteStorage {
         for q in create_tables {
             con.execute(q, []).context("Creating table")?;
         }
-        // At this point the DB schema is that of TaskChampion 0.7.0.
+
+        // -- At this point the DB schema is that of TaskChampion 0.7.0.
```

Comment:
Should all the "0.7.0" version references (also appearing as "0_7_0") be bumped to "0.8.0"?